go/database/sql/go.sum

```diff
@@ -6,6 +6,12 @@ github.com/google/sqlcommenter/go/core v0.0.5-beta h1:axqYR1zQCCdRBLnwr/j+ckllBS
 github.com/google/sqlcommenter/go/core v0.0.5-beta/go.mod h1:GORu2htXRC4xtejBzOa4ct1L20pohP81DFNYKdCJI70=
 github.com/google/sqlcommenter/go/core v0.1.0 h1:g5jL8HUk2Ko9mMPoCI4jP47dyJhpNyu6E8WmiEhh8OU=
 github.com/google/sqlcommenter/go/core v0.1.0/go.mod h1:GORu2htXRC4xtejBzOa4ct1L20pohP81DFNYKdCJI70=
+github.com/google/sqlcommenter/go/core v0.1.1 h1:DXaWbnQHJ00RqqGiyySPnaNHLqWuemtdP1DIheVXa/A=
+github.com/google/sqlcommenter/go/core v0.1.1/go.mod h1:GORu2htXRC4xtejBzOa4ct1L20pohP81DFNYKdCJI70=
+github.com/google/sqlcommenter/go/core v0.1.2-beta h1:fDg7rYUmqO1UJ3SqvDDEB7Wc0kDjpzgS81VGhcJTkoI=
```

Comment:
can you see why this beta dependency is there?

Reply:
Done. Ran go mod tidy.